discovery-provider/alembic/versions/8c10373f2615_user_tips.py

```diff
@@ -0,0 +1,73 @@
+"""User Tips
+
+Revision ID: 8c10373f2615
+Revises: 2cfd17a60647
+Create Date: 2022-04-14 00:06:55.572866
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from src.utils.alembic_helpers import table_exists
+
+# revision identifiers, used by Alembic.
+revision = "8c10373f2615"
+down_revision = "2cfd17a60647"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # USER TIPS
+    if not table_exists("user_tips"):
+        op.create_table(
+            "user_tips",
+            sa.Column("slot", sa.Integer(), nullable=False),
+            sa.Column("signature", sa.String(), nullable=False),
+            sa.Column("sender_user_id", sa.Integer(), nullable=False),
+            sa.Column("receiver_user_id", sa.Integer(), nullable=False),
+            sa.Column("amount", sa.BigInteger(), nullable=False),
+            sa.Column("created_at", sa.DateTime(), nullable=False),
+            sa.PrimaryKeyConstraint("slot", "signature"),
```

Comment:
Any particular reason to have pkey be combo of slot + signature? Seems like if we just did signature, we coiuld avoid having the extra index?

Reply:
Well, if we did just signature, we'd have to have an index on slot anyway. Although I too think that signature is sufficiently unique, so maybe I'll do this anyway?